crates/edr_provider/src/requests/debug.rs

```diff
@@ -120,3 +121,48 @@ impl From<DebugTraceConfig> for edr_evm::DebugTraceConfig {
         }
     }
 }
+
+/// This is the JSON-RPC Debug trace format
+#[derive(Debug, Clone, serde::Deserialize, serde::Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct RpcDebugTraceResult {
+    pub failed: bool,
+    pub gas: u64,
```

Comment:
Did the old format require a `0x...` representation? What made you decide to go with normal `u64` serialization?

Do you have a reference to support this choice?

If we should support a hex representation of a quantity, you should use `alloy_serde::quantity`.

This applies to all `u64` usages here.

Reply:
Yes so I used `u64` because `gas_used` in `DebugTraceResult` is also `u64`:
https://github.com/NomicFoundation/edr/blob/main/crates/edr_evm/src/debug_trace.rs#L220

Reply:
Noted the point on `alloy_serde::quantity` for hex quantities.